.github/workflows/process-results.yml

```
@@ -0,0 +1,41 @@
+name: Process benchmark results
+
+on:
+  push:
+    branches:
+    #  - main
+  pull_request:
+
+jobs:
+  run-benchmark-monitor:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout processing script
+      uses: actions/checkout@v3
+      with:
+        repository: cz4rs/benchmark_monitor
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        path: benchmark-results
+
+    - name: Process available benchmark results
+      run: |-
+        python3 -m venv env
+        source env/bin/activate
+        pip install -r requirements.txt
+        python benchmark_monitor.py -d ./benchmark-results -o output
+
+    - name: Generate nojekyll file
+      working-directory: output
+      run: touch .nojekyll
+
+    - name: Deploy
+      # if: ${{ github.ref == 'refs/heads/main' }}
```

Only deploy from `main`:
```suggestion
      if: ${{ github.ref == 'refs/heads/main' }}
```